avaframe/out3Plot/outQuickPlot.py

```diff
@@ -122,6 +123,14 @@ def generatePlot(dataDict, avaName, outDir, cfg, plotDict):
     ax3.set_aspect('auto')
     ax3.set_xlabel('x [m]')
     ax3.set_title('Difference ref-sim')
+
+    # for difference histogramm - remove dataDiff == 0 values from array
+    dataDiffPlot = dataDiff[dataDiff != 0.0]
+    axin2 = ax3.inset_axes([0.75, 0.1, 0.25, 0.25])
```

Comment:
Is there way to place this automatically? Similiar to what is available for the legend...